amundsen_application/static/js/components/TableDetail/DetailList/DetailListItem/index.tsx

```diff
@@ -49,6 +51,48 @@ class DetailListItem extends React.Component<DetailListItemProps, DetailListItem
     return moment(unixEpochSeconds * 1000).format("MMM DD, YYYY");
   };
 
+  renderColumnType = (columnIndex: number, type: string) => {
+    const truncatedTypes: string[] = ['array', 'struct', 'map'];
+    let shouldTrucate = false;
+
+    const fullText = type.toLowerCase();
+    let text = fullText;
+
+    truncatedTypes.forEach((truncatedType) => {
+      if (type.startsWith(truncatedType) && type !== truncatedType) {
+        shouldTrucate = true;
+        text = `${truncatedType}<...>`;
+        return;
+      };
+    })
+
+    if (shouldTrucate) {
+      const popoverHover = (
+        <Popover className='column-type-popover' id={`column-type-popover:${columnIndex}`}>
+          {fullText}
+        </Popover>
+      );
+      const stopPropagation = (event) => {
+        event.stopPropagation();
+      }
+      return (
+        <OverlayTrigger
+          trigger={['click']}
+          placement='right'
+          overlay={popoverHover}
+          rootClose={true}>
```

Comment:
Can we try using `delayHide` for this element? I think when users try to highlight they might move their mouse outside the hoverable zone which could hide the tooltip. I'm not sure if the hide event will get debounced/cancelled if the user moves the mouse back into the hoverable zone, but it's worth investigating.

Reply:
Oh this is click only. I see.